pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -112,6 +112,8 @@
 , withXcbShape ? withFullDeps # X11 grabbing shape rendering
 , withXcbShm ? withFullDeps # X11 grabbing shm communication
 , withXcbxfixes ? withFullDeps # X11 grabbing mouse rendering
+, withXevd ? withFullDeps && lib.versionAtLeast version "7" # MPEG-5 EVC decoding
+, withXeve ? withFullDeps && lib.versionAtLeast version "7" # MPEG-5 EVC encoding
```

Comment:
Given these rely on unstable versions, I think I'd prefer them off by default for now.

Reply:
Switch over to stable version with patches, do you still think that they should be switched off in that case?

Reply:
Nope, should be fine in that case. EVC support is one of the main features of ffmpeg v7, so we should ideally have this enabled.